distribution/ddf/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -21,7 +21,6 @@
 # Comma separated list of features repositories to register by default
 #
 featuresRepositories= \
- mvn:org.codice.ddf/kernel/${project.version}/xml/features, \
```

Comment:
❓  Would you also need to get rid of kernel as a boot feature here or no?

Reply:
I still want to start the `kernel` feature which is now defined in the platform-app features repository. This PR deletes the kernel features repository.

Reply:
Ahh okay, confusing.